api/src/main/resources/jakarta/xml/ws/wsdl_customizationschema_2_0.xsd

```diff
@@ -79,7 +79,7 @@
                 <xs:choice minOccurs="0" maxOccurs="unbounded">
                     <xs:group ref="jaxws:declaration"/>
                     <xs:element ref="jaxws:bindings"/>
-                </xs:choice>               
```

Comment:
Gratuitous whitespace changes only?

Reply:
Yes. I was normalizing whitespaces in other schemas (there were few tabs instead of whitespaces, trimming lines) and this one felt into to bucket... I can avoid this change in this file if you want as I see no harm in keeping them in this case.

Reply:
Your choice, but I prefer not to make *any* changes in already published schemas files.

Reply:
ok